tests/ModLoaderTests.cpp

```diff
@@ -92,12 +92,14 @@ TEST(ModLoaderTests, BrokenMissingAndNonexistentModsAreDiscarded)
 
 	ASSERT_EQ(mods.size(), 1);
 	EXPECT_EQ(mods[0].name, "The Mod");
-	EXPECT_EQ(mods[0].path, "GameDocumentsFolder/mod/themodsfolder/");
+	EXPECT_EQ(mods[0].path.string(), "GameDocumentsFolder\\mod\\themodsfolder");
```

Comment:
I think forward slashes are needed on Linux and Mac. Since Windows supports them as well, it'd probably be best to use them, if possible.

Reply:
The string() function localizes them to OS-appropriate slashes automatically.